packages/flutter_tools/test/src/fake_process_manager.dart

```diff
@@ -0,0 +1,206 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
This file needs comments.

Reply:
done

Reply:
Cool, thanks. This is a really nice API, but it doesn't appear to have any non-trivial uses in this PR. What are your plans for it?

Reply:
My first customer is https://github.com/flutter/flutter/pull/42491. I suspect the API will evolve as it gets more users.